src/components/PersonalData.js

```diff
@@ -15,13 +15,23 @@ import "../login/styles/index.scss";
 
 /* FORM */
 
-const validate = (values) => {
+const validatePersonalData = (values) => {
   const errors = {};
+  const spacePattern = /^\s+$/;
+  const onlyLettersPattern = /^[a-zA-ZäöåÄÖÅ]+(\s+[a-zA-ZäöåÄÖÅ]+)*$/;
+  
   ["given_name", "surname", "display_name", "language"].forEach((pdata) => {
     if (!values[pdata]) {
       errors[pdata] = "required";
     }
-  });
+    else if(spacePattern.test(values[pdata])){
+      errors[pdata] = "blank space not allowed";
```

Comment:
yes! however not in the beginning of the text.

Reply:
yes, I literally meant `.trim()` and we actually don't need to alert the user with a message about it we can just make sure to trim any input. We should do this instead of `const spacePattern = /^\s+$/;`.

Reply:
I think we don't need to add a trim due to the input datas will be added without space.